node/silkworm/common/secp256k1_context.hpp

```diff
@@ -36,10 +37,32 @@ class SecP256K1Context final {
     // escape hatch
     secp256k1_context* raw() { return context_; }
 
-    bool verify_private_key_data(const ByteView& data) {
+    [[nodiscard]]
+    bool verify_private_key_data(const ByteView& data) const {
         return secp256k1_ec_seckey_verify(context_, data.data());
     }
 
+    bool create_public_key(secp256k1_pubkey* public_key, const ByteView& private_key) const {
+        return secp256k1_ec_pubkey_create(context_, public_key, private_key.data());
+    }
+
+    Bytes serialize_public_key(const secp256k1_pubkey* public_key, bool is_compressed) const {
+        size_t data_size = is_compressed ? 33 : 65;
```

Comment:
Please use two constants here with [this format](https://google.github.io/styleguide/cppguide.html#Constant_Names) instead of magic numbers to improve the code readability.